File: doc/_i18n/en/tutorials/introduction/dual-arm-controller/add_kinova_tasks.cpp
// In the header
std::shared_ptr<mc_tasks::PostureTask> kinovaPostureTask_;
std::shared_ptr<mc_solver::KinematicsConstraint> kinovaKinematics_;
// In the reset function
kinovaPostureTask_ = std::make_shared<mc_tasks::PostureTask>(solver(), 1);
solver().addTask(kinovaPostureTask_);
kinovaKinematics_ = std::make_shared<mc_solver::KinematicsConstraint>(robots(), 1, solver().dt());
solver().addConstraintSet(*kinovaKinematics_);

File: doc/_i18n/jp/tutorials/introduction/dual-arm-controller/add_kinova_tasks.py
# resetコールバック関数内
self._kinovaPostureTask = mc_tasks.PostureTask(
    self.qpsolver, 1, 5.0, 1000.0
)
self._kinovaKinematicsConstraint = mc_solver.KinematicsConstraint(
    self.robots(), 1, self.qpsolver.dt()
)
self.qpsolver.addTask(self._kinovaPostureTask)
self.qpsolver.addConstraintSet(self._kinovaKinematicsConstraint)